corehq/form_processor/management/commands/archive_all_forms_for_user_in_domain.py

```diff
@@ -0,0 +1,60 @@
+from __future__ import absolute_import
+from __future__ import unicode_literals
+from __future__ import print_function
+
+from io import open
+
+from django.core.management.base import BaseCommand
+
+from casexml.apps.case.cleanup import rebuild_case_from_forms
+from corehq.apps.users.models import CouchUser
+from corehq.form_processor.interfaces.dbaccessors import FormAccessors
+from corehq.form_processor.models import RebuildWithReason
+from casexml.apps.case.xform import get_case_updates
+from corehq.util.log import with_progress_bar
+from corehq.form_processor.interfaces.processor import FormProcessorInterface
+
+
+class Command(BaseCommand):
+    help = """
+        Bulk archive forms for user on domain.
+        First archive all forms and then rebuild corresponding cases
+    """
+
+    def add_arguments(self, parser):
+        parser.add_argument('user_id')
+        parser.add_argument('domain')
+
+    def handle(self, user_id, domain, **options):
+        user = CouchUser.get_by_user_id(user_id)
+        form_accessor = FormAccessors(domain)
+
+        # ordered with latest form's id on top
+        form_ids = form_accessor.get_form_ids_for_user(user_id)
+        forms = [f for f in form_accessor.get_forms(form_ids) if f.is_normal]
+        print("Found %s normal forms for user" % len(form_ids))
+
+        case_ids_to_rebuild = set()
+        for form in forms:
+            form_case_ids = set(cu.id for cu in get_case_updates(form))
```

Comment:
This will not get forms that only have ledger updates for a case. I think the previous version was better.

Reply:
reverting this change.